src/main/java/io/nats/client/Subscription.java

```diff
@@ -41,12 +42,12 @@
     /**
      * @return the subject associated with this subscription, will be non-null
      */
-    public String getSubject();
+    public ByteBuffer getSubject();
```

Comment:
This might break existing applications?  ...String -> ByteBuffer

Reply:
Changed to return a `String`.